README.md

```diff
@@ -232,12 +232,12 @@ macro, have a look at the macro implementation. No magic there.
 
 The following instructions should result in a functional extension, loaded into REAPER on start:
 
-1. Run `cargo new reaper-my-extension --lib` to initialize the project
-2. Run `cargo build` from within `reaper-my-extension` to generate the compiled plugin extension inside of the `target/debug` directory
+1. Run `cargo new reaper_my_extension --lib` to initialize the project
```

Comment:
Is snake case necessary here? I think Cargo will take care of using an underscore in the lib file name even if you use dashes here.

Reply:
OH 🤦 Didn't know that -- in that case no, not necessary at all!